app/controllers/referee_interface/reference_controller.rb

```diff
@@ -0,0 +1,47 @@
+module RefereeInterface
+  class ReferenceController < ActionController::Base
+    layout 'application'
+
+    def feedback
+      reference = Reference.find_by(token: params[:token])
+
+      if reference.present?
+        @reference_form = RefereeInterface::ReferenceForm.build_from_reference(reference)
+        @application = reference.application_form
+      else
+        render_404
+      end
+    end
+
+    def submit_feedback
```

Comment:
We're going to have to check if the feedback has already been given, otherwise we're allowing edits forever, which might lead to weirdness.

Reply:
Good point, I've added in my todo list.

Reply:
I've added a condition on the `reference_controller#feedback` to check if the feedback already exists.